packages/shared/src/hooks/referral/useReferralConfig.ts

```diff
@@ -0,0 +1,37 @@
+import { Author } from '../../graphql/comments';
+import { ReferralCampaignKey } from './useReferralCampaign';
+import { CampaignConfig } from '../../graphql/features';
+
+interface UseReferralConfigProps {
+  campaign: string;
+  referringUser: Author;
+}
+
+export const useReferralConfig = ({
+  campaign,
+  referringUser,
+}: UseReferralConfigProps): CampaignConfig => {
+  const getDefaultTitle = () =>
+    `${referringUser.name} invites you to use daily.dev`;
+  const getDefaultDescription = () =>
+    `daily dev is a professional network for developers to learn, collaborate, and grow together. Developers come to daily.dev to discover a wide variety of professional knowledge, create groups where they can collaborate with other developers they appreciate, and discuss the latest trends in the developer ecosystem.`;
+  const getDefaultImages = () => [
+    { url: `https://og.daily.dev/api/refs/${referringUser.id}` },
+  ];
```

Comment:
Why not just raw values instead of functions?

Reply:
Ah, I moved this from a generic function to a hook. Yep, probably we can go with just raw values now since we have all the information already.